showroom/webpack.base.babel.js

```diff
@@ -11,7 +12,12 @@ const paths = {
 module.exports = {
 
   plugins: [
-    new webpack.ContextReplacementPlugin(/moment[\/\\]locale$/, /de|fr|en/)
+    new webpack.ContextReplacementPlugin(/moment[\/\\]locale$/, /de|fr|en/),
+    new StyleLintPlugin({
+      files: '**/*.s?(a|c)ss',
```

Comment:
as we're checking only `scss` syntax (`syntax: 'scss'`) couldn't this be simplified to `**./*.scss`?